packages/core-commands/src/site-editor-navigation-commands.js

```diff
@@ -338,6 +339,24 @@ function useSiteEditorBasicNavigationCommands() {
 			},
 		} );
 
+		result.push( {
+			name: 'core/manage-reusable-blocks',
```

Comment:
```suggestion
			name: 'core/edit-site/open-patterns',
```

There are no hard and fast rules regarding this property, but it is a good idea to make it consistent with other commands.